frontend/src/pages/AgreementPage.tsx

```diff
@@ -0,0 +1,62 @@
+import { Breadcrumbs, Button, Link, Stack, Typography, Chip } from '@mui/material';
+import { GroupContext } from 'contexts/group';
+import React, { FC, useContext } from 'react';
+import SummarizeOutlinedIcon from '@mui/icons-material/SummarizeOutlined';
+import VisibilityOutlinedIcon from '@mui/icons-material/VisibilityOutlined';
+import { useParams } from 'react-router-dom';
+
+const AgreementPage: FC = () => {
+  const { activeAgreements } = useContext(GroupContext);
+  const { agreementId } = useParams();
+  const currentAgreement = activeAgreements.filter(
+    (agreement) => agreement.id.toString() === agreementId
+  )[0];
+  const currentCategory = currentAgreement.category.name;
+  const handleClick = (e: any) => {
+    e.preventDefault();
+    console.log('clicked breadcrumb');
+  };
+  return (
+    <Stack direction="column">
+      <Breadcrumbs separator="›" aria-label="breadcrumb">
+        <Link underline="hover" key="1" color="inherit" href="/" onClick={handleClick}>
+          {currentCategory || 'categoryName'}
+        </Link>
+        <Link underline="hover" key="1" color="inherit" href="/" onClick={handleClick}>
+          <Stack direction="row" gap="5px">
+            <SummarizeOutlinedIcon />
+            {localStorage.getItem('agreementName') || 'Agreement Name'}
```

Comment:
@yishain11 match zeplin `font-size` and `font-weight`

Reply:
done